Sources/FluentTester/Tester+SoftDelete.swift

```diff
@@ -56,24 +56,16 @@ extension Tester {
             throw Error.failed("Water was not forced deleted")
         }
 
-        try ethanol.save()
-
-        guard try Compound.count() == 1 else {
-          throw Error.failed("Compound count did not equal 4.")
-        }
-
         try Compound.makeQuery().delete()
 
         guard try Compound.count() == 0 else {
           throw Error.failed("Compound count did not equal 0 after delete.")
         }
 
-        guard try Compound.withSoftDeleted().count() == 1 else {
+        guard try Compound.withSoftDeleted().count() == 3 else {
           throw Error.failed("Water was not soft deleted")
         }
 
-        try water.restore()
-
         try Compound.makeQuery().forceDelete()
 
         guard try Compound.withSoftDeleted().count() == 0 else {
```

Comment:
One thing that this doesn't test is whether `forceDelete` deletes non-soft deleted instance. Is it okay to assume that if it deletes `softDeleted` instances it will delete others too or shall I add another test for it?